app/schema/widgets/date_widget.py

```diff
@@ -49,14 +48,17 @@ def _get_day_field(self, day):
                   },
                 }
 
-    def _get_month_field(self, month):
+    def _get_month_field(self, month, is_mandatory):
 
         return {
                 'label': {
                     'for': self.name + '-month',
                     'text': 'Month',
                 },
                 'select': {
+                    'mandatory': is_mandatory,
+                    'placeholder': 'Select month',
+                    'selected': bool(month),
```

Comment:
Think we need to do this across the board, as all dates, labels etc are all non translated at the minute, I reckon a new story